PokemonTabletopAdventures_v3/PTA3.html

```diff
@@ -1388,6 +1499,370 @@ <h2>Configuration Options</h2>
         <textarea name="attr_json_export" title="Attribute: json_export" type="text"></textarea>
       </div>
     </div>
+    <h2>Form Changes</h2>
+    <div class="sheet-text-background">
+      <p class="sheet-dark-text">
+        Configure form changes here. For each form, there is a box to enable its use. Checking the box allows you to
+        configure the form, and adds a button to the top-right of the main Character page.
+        Click that button to activate and end the form change.
+      </p>
+      <p class="sheet-dark-text">
+        Note: Type changes are prioritized in the following order: Tera, Mega, Custom1, Custom2, Custom3. For example, if you have Mega and Custom1
+        active at the same time, your type will be your Mega type.
+      </p>
+    </div>
+    <div class="sheet-configuration-grid sheet-extra-row-gap">
+      <div class="sheet-tera-block sheet-text-background">
+        <input class="sheet-enable-tera" name="attr_enable-tera" type="checkbox" hidden />
+        <h3><label class="sheet-afflictions-text"><input class="sheet-enable-tera" name="attr_enable-tera" type="checkbox" /> Terastallization</label></h3>
```

Comment:
`sheet-afflictions-text` nets us a `font-size: 1em` that is over-written in dark mode, and is applied in light mode even without this CSS rule.

Reply:
I recommend making a new rule here anyway - it would give us the opportunity to make the headers all have the same size when minimised;
<img width="451" alt="Screenshot 2023-02-23 at 17 28 08" src="https://user-images.githubusercontent.com/25342733/220983875-e5780620-16e4-4e86-ada3-23810848f463.png">

using the following rules;
```CSS
.charsheet label.sheet-form-change-header {
    font-size: 1em;
    height: 24px;
    padding-top: 8px;
}

.charsheet label.sheet-form-change-header input.sheet-custom-move-name {
    padding-top: 0px;
    margin-top: -4px;
}
```

It's not the neatest way of shuffling the inputs back up, but it works, and it means we have fixed-height headers. Your call on this one, though.